File: Objective-C/FlickrSearch/FSFavoriteVC.m
//
//  FSFavoriteVC.m
//  FlickrSearch
//
//  Created by Alfred Yu on 2020/10/21.
//

#import "FSFavoriteVC.h"
#import "DataManager.h"

@interface FSFavoriteVC ()

@end

@implementation FSFavoriteVC

- (instancetype) init
{
    NSMutableArray *list = [DataManager queryRecord];
    
    self = [super initWithPhotoList:list nibName:@"FSResultVC" bundle:nil];
    
    self.str_notFound = @"Empty !!";
    
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
}

@end

File: Swift/FlickrSearch/FSFavoriteVC.swift
//
//  FSFavoriteVC.swift
//  FlickrSearch
//
//  Created by Alfred Yu on 2020/10/21.
//

import UIKit

class FSFavoriteVC: FSResultVC
{
    override func viewDidLoad()
    {
        super.viewDidLoad()
    }
    
    init()
    {
        super.init(PhotoList: DataManager.queryRecord(), nibName: "FSResultVC", bundle: nil)
        str_notFound = "Empty !!";
    }
    
    required init?(coder: NSCoder)
    {
        fatalError("init(coder:) has not been implemented")
    }
    
    
}
